docs/source/release/v6.0.0/diffraction.rst

```diff
@@ -29,6 +29,7 @@ New features
 Improvements
 ############
 - :ref:`WANDPowderReduction <algm-WANDPowderReduction>` now accepts a sequence of input workspaces, combining them to reduce to a single spectrum.
+- :ref:`PowderILLDetectorScan <algm-PowderILLDetectorScan>` is fixed when treating more than one scans merged.
```

Comment:
Can you rephrase to make clear it is about a) Multiple scans merged into one file, b) 'fixed' means corrected? For example 'fixed issue when treating merging multiple scans'.